.github/workflows/scripts/addClosingMessage.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @param {Object} params
+ * @param {import("@actions/core")} params.core
+ * @param {ReturnType<import("@actions/github").getOctokit>} params.github
+ * @param {import("@actions/github").context} params.context
+ */
```

Comment:
```suggestion
// @ts-check

/**
 * @param {Object} params
 * @param {import("@actions/core")} params.core
 * @param {ReturnType<import("@actions/github").getOctokit>} params.github
 * @param {import("@actions/github").context} params.context
 */
```

Probably a good idea to allow TS to check the files, so we can prevent some issues.